test/module/irohad/torii/command_sync_client_test.cpp

```diff
@@ -39,13 +37,18 @@ class CommandSyncClientTest : public testing::Test {
  * @then the same method of the server is called and client successfully return
```

Comment:
Description is outdated. Maybe it should mention that data is passed and returned from stub is correct. Please check other tests as well